packages/tools/src/stateManagement/segmentation/polySeg/polySegConverters.js

```diff
@@ -0,0 +1,387 @@
+import { expose } from 'comlink';
+import vtkImageData from '@kitware/vtk.js/Common/DataModel/ImageData';
+import vtkDataArray from '@kitware/vtk.js/Common/Core/DataArray';
+import ICRPolySeg from '@icr/polyseg-wasm';
+import { utilities } from '@cornerstonejs/core';
+import { getBoundingBoxAroundShapeWorld } from '../../../utilities/boundingBox';
+import { pointInShapeCallback } from '../../../utilities';
+import { isPointInsidePolyline3D } from '../../../utilities/math/polyline';
+
+const obj = {
```

Comment:
Can we call this something else - like polySegConverters

Reply:
Also, would like a bit of docs here saying this is an exposed set of objects shared between webworker and primary contexts,  `@see ....` so that there is a bit of help to someone new getting into this framework.  If I hadn't seen your earlier reviews, I would be COMPLETELY clueless as to what you were doing here.